drivers/hmis/extensions/grda_warehouse/hud/client_extension.rb

```diff
@@ -14,6 +16,12 @@ module ClientExtension
       has_many :hmis_custom_assessments, through: :enrollments
       has_many :hmis_source_custom_assessments, through: :source_enrollments, source: :hmis_custom_assessments
       has_many :custom_services, through: :source_enrollments
+      has_many :ce_client_proxies, as: :client, class_name: 'Hmis::Ce::ClientProxy', dependent: :destroy
+      has_many :ce_match_candidates, class_name: 'Hmis::Ce::Match::Candidate', through: :ce_client_proxies
+
+      # HMIS source clients represented as Hmis::Hud::Client
+      # (Normal source_clients are represented as GrdaWarehouse::Hud::Client)
+      has_many :hmis_source_clients, -> { hmis }, class_name: 'Hmis::Hud::Client', through: :warehouse_client_destination, source: :source
```

Comment:
Does adding ` -> { hmis }` scope here make sense?
- for now (mvp) I think it won't have any negative effect, because in the CeCandidate schema object, we filter source enrollments to only those the current data source, anyway
- I added it because it felt weird to return non-hmis clients as `Hmis::Hud::Client`s.